### PR TITLE
adds Dockerfile and build workflow for CSV provider

### PR DESCRIPTION
**Pull request by eriksven**

### CODE REVIEW

**Review by lukasmittag — CHANGES_REQUESTED** [review]
See comments

**Comment by eriksven** [comment]
@lukasmittag thank you for your feedback. I updated the PR accordingly.

File: csv_provider/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+ENV PATH="/opt/venv/bin:$PATH"
+
+RUN /opt/venv/bin/python3 -m pip install --upgrade pip \
+    && pip3 install --no-cache-dir -r requirements.txt
+
+
+RUN pip3 install wheel scons && pip3 install pyinstaller
+
+# By default we use certificates and tokens from kuksa_certificates, so they must be included
```
Comment (lukasmittag) [inline csv_provider/Dockerfile:37]: delete comment as well

**Review by lukasmittag — CHANGES_REQUESTED** [review]
just one minor change :)

File: csv_provider/Dockerfile (hunk 1 above)
Comment (eriksven) [reply csv_provider/Dockerfile]: Done

**Review by lukasmittag — APPROVED** [review]

**Pull request merged**